modules/tinymce/src/core/test/ts/browser/keyboard/ArrowKeysInlineBoundariesTest.ts

```diff
@@ -336,4 +336,27 @@ describe('browser.tinymce.core.keyboard.ArrowKeysInlineBoundariesTest', () => {
       TinyAssertions.assertCursor(editor, [ 0, 0 ], 0);
     });
   });
+
+  context('Arrow keys on inline boundary node when selection is not collapsed', () => {
```

Comment:
This is quite verbose and just duplicates the information (e.g. we already know this test is about arrow keys on inline boundaries). As such, maybe it should be something like this (while trying to keep it similar to what's already there)?
```suggestion
  context('Arrow keys when selection is not collapsed', () => {
```